kernel/monitor.c

```diff
@@ -31,7 +31,7 @@ int stack_backtrace()
 
 	// Your code here.
 	for (u64* fp = (u64*)read_fp(); fp; fp = (u64*)*fp) {
-		printk("  LR %lx  FP %lx  Args %lx %lx %lx %lx %lx\n", *(fp + 1), *fp,
+		printk("  LR %lx  FP %lx  Args %lx %lx %lx %lx %lx\n", *(u64 *)( (u64)*fp + 8), *fp,
```

Comment:
```suggestion
		printk("  LR %lx  FP %lx  Args %lx %lx %lx %lx %lx\n", *(u64 *)(*fp + 8), *fp,
```
`fp` 已经是 `u64 *`，所以 `*fp` 是 `u64` 不用转型了